schema/dereferenced-release-schema.json

```diff
@@ -3087,6 +3087,18 @@
         "eligibilityCriteria": {
           "title": "Eligibility criteria",
           "description": "A description of any eligibility criteria for potential suppliers.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "deprecated": {
+            "description": "This field is deprecated in favor of exclusionGrounds, which uses more standard terminology.",
```

Comment:
```suggestion
            "description": "This field had unclear semantics and used ambiguous terms, and is deprecated in favor of exclusionGrounds.",
```

Reply:
New issue in the style guide for deprecated.description fields: https://github.com/open-contracting/standard-development-handbook/issues/241

Thanks!